app/views/idv/sessions/index.html.slim

```diff
@@ -24,7 +24,7 @@
       class: 'block col-12 field', input_html: { 'aria-labelledby': 'label-dob' }
   h2#label-ssn.heading = t('idv.titles.session.ssn')
   .mb4.sm-col-5
-    = f.input :ssn, label: false, required: true, pattern: '[0-9]*',
+    = f.input :ssn, label: false, required: true, pattern: '^\d{3}-?\d{2}-?\d{4}$',
```

Comment:
@sbc100 Where would you suggest those patterns be defined?  Also it doesn't look like we are doing any server side validation of those particular fields at the moment - I can follow up w/ Peter on that though, perhaps the same pattern can be shared for both purposes.